evalml/model_understanding/graphs.py

```diff
@@ -293,113 +293,9 @@ def graph_roc_curve(y_true, y_pred_proba, custom_class_names=None, title_additio
     return _go.Figure(layout=layout, data=graph_data)
 
 
-def _calculate_permutation_scores_fast(pipeline, precomputed_features, y, objective, col_name,
```

Comment:
I moved everything to a permutation_importance file since we were starting to add more methods that were just to calculate permutation importance. Left the graphing permutation importance method here, though.